src/zc/table/fieldcolumn.py

```diff
@@ -103,20 +107,20 @@ def getInputWidget(self, item, formatter):
             widget = component.getMultiAdapter((field, request), iface)
         else:
             widget = form_field.custom_widget(field, request)
-        if form_field.prefix: # this should not be necessary AFAICT
+        if form_field.prefix:  # this should not be necessary AFAICT
             prefix = '%s.%s' % (prefix, form_field.prefix)
         widget.setPrefix(prefix)
         return widget
 
     def getRenderWidget(self, item, formatter, ignore_request=False):
         widget = self.getInputWidget(item, formatter)
         if (ignore_request or
-            IDisplayWidget.providedBy(widget) or
-            not widget.hasInput()):
+                IDisplayWidget.providedBy(widget) or
+                not widget.hasInput()):
             widget.setRenderedValue(self.get(item, formatter))
         return widget
 
-    ###### subclass customization API ######
+    # subclass customization API ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..